src/components/Staking/StakingProductsCardGrid/StakingProductCard.tsx

```diff
@@ -29,6 +29,7 @@ const getIconFromName = (
 ): ComponentType<SVGProps<SVGElement>> => {
   const {
     [imageName + "GlyphIcon"]: Icon,
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
As far as eslint is concerned, [using `require` in  pretty much banned](https://typescript-eslint.io/rules/no-var-requires/).

Possible workaround here is to run a call in `useEffect` and storing the import in state. This would use the more modern `import()` to handle dynamic importing, which returns a promise.

```tsx
const [Svg, setSvg] = useState<ChakraIconType | null>(null)

useEffect(() => {
  const getIconFromName = async (imageName: string) => {
    const Icon: ChakraIconType = await import(
      "@/components/icons/staking"
    ).then((icons) => icons[imageName + "GlyphIcon"])
    setSvg(Icon)
  }

  getIconFromName(imageName)
})
```

Caveat here is now that there is essentially a fetch occurring, there is a delay in the render of the icons.